roles/python-3/tasks/main.yml

```diff
@@ -8,3 +8,8 @@
   - name: install Python 3
     include_tasks: install.yml
     when: python_3_installed_version.rc != 0 or python_3_installed_version.stdout != python_3_version_output
+
+  - name: add python3 as priority alternate
+    shell:
+      cmd: "update-alternatives --install /usr/bin/python python /usr/bin/python2.7 1 && update-alternatives --install /usr/bin/python python /usr/bin/python3 2"
```

Comment:
What does the last argument `2` mean?

Reply:
its the priority. python3 `2` has a higher priority than python2 `1`.

Reply:
Sounds good. Can you also put this in the comment?